jigglypuffRL/deeprl/dqn/dqn.py

```diff
@@ -91,6 +92,7 @@ def __init__(
         self.save_model = save_model
         self.save = save_params
         self.load = load_params
+        self.pretrained = pretrained
         self.checkpoint = self.__dict__
```

Comment:
Can you add a `get_hyperparams` for this? Instead of `self.__dict__`. Using `self.__dict__` creates problems mentioned in #60

Reply:
If we use `get_hyperparams` in `__init__`, it'll give an error since `self.model` is not initialized yet. How do you think we should go about it then?

Reply:
I'll add a parameter `include_model` in get_hyperparams which includes the model weights only if True

Reply:
We dont really need it `__init__`. We can do it when `learn` is called? Or when its trying to save it

Reply:
We dont really need the call to `get_hyperparams` in constructor, no?

Reply:
Yeah, true. I've changed it now